packages/@romejs/core/master/linter/CompilerLinter.ts

```diff
@@ -78,7 +78,7 @@ export default class CompilerLinter {
                 printer.addDiagnostic({
                   category: '',
                   message:
-                    "Files excluded from linting as it's not enabled. Add `lint: {enabled: true}`",
+                    'Files excluded from linting as it's not enabled. Add `lint: {"enabled": true}`',
```

Comment:
Oops, looks like the apostrophe wasn't escaped.

```suggestion
                    'Files excluded from linting as it\'s not enabled. Add `lint: {"enabled": true}`',
```